src/Shared/Grpc/ProtoUtils.cs

```diff
@@ -223,18 +223,21 @@ internal static Timestamp ToTimestamp(this DateTime dateTime)
     /// <param name="instanceId">The orchestrator instance ID.</param>
     /// <param name="customStatus">The orchestrator customer status or <c>null</c> if no custom status.</param>
     /// <param name="actions">The orchestrator actions.</param>
+    /// <param name="completionToken">The completion token from the work item.</param>
```

Comment:
nit: Could we add more details here? Like the completion token generated from the DTS backend, as we don’t have this information for other backends.

Reply:
I updated the comment here to provide a bit more context. Note that it's not really a "backend" concept per-se, but more of a remote durable task engine (which we sometimes call the "sidecar") concept. For example, the implementation could be different for Dapr or for DTS, and it doesn't really matter what kind of backend store is used.